{% set ref = 'no' %}
{% extends "layout.html" %}
  
{% block title %}Results{% endblock %}
{% block lead %} iGEM-IIT-Delhi page for Results. {% endblock %}

  

{% block page_content %}


<h2>Lichenysin gene cluster</h2>
<hr>
<p><b>Preparation of vectors: </b>
  Plasmid, pSIM7 was isolated from its culture. The vector was digested with SmaI and BglII, having a fallout of about 1816 bp, after which the linearized band was cut and gel extraction was performed with it. Gel extraction of the linear band was performed and then eluted. All of the gels were run on 1% agarose.
  <br></p>

  <div>
    <img src="https://github.com/Ravinder210/igem_results_img/blob/main/1result_lichenysin_gene_cluster_preparation%20of%20vectors.png?raw=true" alt=" wait for image to be uplaoded"  width="200" height="200">

  </div>

<p><b>Preparation of insert:</b>

  As the length of the insert to be incorporated was 26.6 Kb, special primers with overhangs were designed so that specific binding can take place. Unfortunately, even after multiple attempts of optimizing temperature, template concentrations with both colony PCR and PCR of isolated genomic DNA of B. licheniformis an amplicon could not be obtained. </p>

  <div>
    <img src="https://github.com/Ravinder210/igem_results_img/blob/main/2result_lichenysin_gene_cluster_preparation%20of%20insert.png?raw=true" alt=" wait for image to be uplaoded"  width="200" height="200">

  </div>
<br>
<h2>Dual Promoter System
</h2>
<hr>
<p> <b> 
  Preparation of inserts:
  </b> <br>
  We revived the synthesized genes for alasan, and rhlAB, and made working stocks of 100ng/ul. We also revived their primers and made working stocks of those to use in our PCR reactions. The gene for Ptet was obtained by amplification of the 156bp fragment from a plasmid carrying the promoter. We obtained amplification of all the three gene fragments we set out to ultimately clone, Ptet, aln, and rhlAB with Taq polymerase and using the respective annealing temperatures as was found out from the NEB Tm calculator, 52¬∞C, 51¬∞C and 62¬∞C respectively with extension times as was dictated by the speed of the Taq polymerase we used as well as the gene length. For insert digestion we then digested our gel extracted PCR products with the sets of enzymes we needed to clone them with. The gene for alasan, aln was digested with NdeI and HindII, Ptet with BglII and SphI, and rhlAB was digested with XbaI and SphI. The reactions were digested for 8 hours to 12 hours depending on the concentration of the gel extracted elute. They were then PCR purified and ran on 1% agarose gel. The images have been added below:
  
  </p> 
  <div>
    <img src="https://github.com/Ravinder210/igem_results_img/blob/main/3result_dual%20promoter%20system_preparation%20of%20inserts.png?raw=true" alt=" wait for image to be uplaoded"  width="200" height="200">

  </div>  
  <div>
    <img src="https://github.com/Ravinder210/igem_results_img/blob/main/4result_dual%20promoter%20system_preparation%20of%20inserts2.png?raw=true" alt=" wait for image to be uplaoded"  width="200" height="200">

  </div>
<p> <b>Preparation of vectors:</b> Plasmids, pTAC (for cloning), and TBS7-Ptet-X-deGFP_Trps16,(for Ptet) were isolated from their cultures. After initial optimization runs, we prepared two different vectors at first. The vector to clone aln, pTAC, was digested with NdeI and HindII, having a fallout of about 500bp, after which the linearized band was cut and gel extraction was performed with it. The vector for Ptet was made by sequentially digesting pTAC plasmid with SphI and BglII and the linear band was cut and eluted, seeing a fall out of 200bp. After clone confirmation of Ptet, we immediately constructed the vector for cloning rhlAB as well, by digesting the clone with XbaI and SphI. Gel extraction of the linear band was performed and then eluted. All of the gels were run on 1% agarose.
 <br>
  
 <div>
  <img src="https://github.com/Ravinder210/igem_results_img/blob/main/5result_dual%20promoter%20system_preaparation%20of%20vector.png?raw=true" alt=" wait for image to be uplaoded"  width="200" height="200">

</div>
<div>
  <img src="https://github.com/Ravinder210/igem_results_img/blob/main/6result_dual%20promoter%20system_preparation%20of%20vector2.png?raw=true" alt=" wait for image to be uplaoded"  width="200" height="200">

</div>

  <br> <b>Ligation, transformation and clone confirmation:</b> For each cloning, respective inserts and vectors were ligated using 1:3 and 1:5 molar ratios and were transformed to freshly prepared E. coli DH5Œ± cells. The colonies that came the next day were screened for colony PCR, using vector specific primers as well as gene specific primers. For alasan, we used Ptac_FP and aln_RP, for Ptet, we used gene specific forward and reverse primers. For rhlAB clones, we used Ptet_FP and rhlAB_RP. The colonies which came positive were selected and inoculated for plasmid isolation. For alasan, we immediately tested the colonies for Emulsification Index assay with a wide variety of oils. The colonies tested positive for EI assay, so we selected one of these to do our downstream work. The plasmids were then used for putting PCR against alasan gene, where all of them tested positive, thereby confirming our clone, Ptac-aln. The idea of cloning our gene under a tac promoter was that we could perform all our assays using the cloning strain, E. coli DH5Œ± cells itself, they bypassing the need to use a specialized expression strain harboring a lysogenized phage expressing T7 RNA polymerase.  The  colony PCR images as well as the clone confirmation images are provided below with a detailed figure legend.

  <div>
    <img src="https://github.com/Ravinder210/igem_results_img/blob/main/7result_dual%20promoter%20system_Ligation,%20transformation%20and%20clone%20confirmation.png?raw=true" alt=" wait for image to be uplaoded"  width="200" height="200">

  </div>
  <div>
    <img src="https://github.com/Ravinder210/igem_results_img/blob/main/8result_dual%20promoter%20system_Ligation,%20transformation%20and%20clone%20confirmation2.png?raw=true" alt=" wait for image to be uplaoded"  width="200" height="200">

  </div>

  <div>
    <img src="https://github.com/Ravinder210/igem_results_img/blob/main/9result_dual%20promoter%20system_Ligation,%20transformation%20and%20clone%20confirmation3.png?raw=true" alt=" wait for image to be uplaoded"  width="200" height="200">

  </div>

 <br> <br> <b> SDS-PAGE to detect the presence of the Aln protein:
</b> To visualize the expression of Aln protein in the heterologous host E. coli DH5ùõÇ, the cultures were induced with IPTG at the OD600 of 0.3-0.6. Samples were collected for 3 hours of induction from the cultures, and an empty vector containing just the promoter was used as a control. 
The cultures were sonicated to analyze the expression of the Alasan protein, the cell pellet was dissolved in 3 ml of sonication buffer and was sonicated using a 3 mm, sonication probe for 10 min with a pulse of  20 secs ON and 30 secs OFF cycle at an amplitude of 35%. The cultures were then centrifuged, the supernatant was collected. The pellet after sonication was also lysed. The samples were prepared by dissolving in laemmli dye containing SDS and ùõÉ-mercaptoethanol, the samples were boiled at 95‚ÑÉ for 10 min. These samples of both supernatant and pellet were run on 12% SDS-PAGE, to understand if the protein was soluble and can fold properly in the sonication buffer used. 
A protein of ~45kDa was observed in the supernatant of the induced sample harboring the clone with Aln gene. The same band was not visualized after staining and destaining in samples from pellet after sonication. This suggested that our protein was properly solubilized and was able to fold properly.
<div>
  <img src="https://github.com/Ravinder210/igem_results_img/blob/main/10result_dual%20promoter%20system_SDS-PAGE%20to%20detect%20the%20presence%20of%20the%20Aln%20protein.png?raw=true" alt=" wait for image to be uplaoded"  width="200" height="200">

</div>
<div>
  <img src="https://github.com/Ravinder210/igem_results_img/blob/main/11result_dual%20promoter%20system_SDS-PAGE%20to%20detect%20the%20presence%20of%20the%20Aln%20protein2.png?raw=true" alt=" wait for image to be uplaoded"  width="200" height="200">

</div>
<br>
<br>
 <b> Assays to detect the type of biosurfactants: </b> The induction of 1mM IPTG (Isopropyl Œ≤-D-1-thiogalactopyranoside) activated the tac promoter under which aln gene has been cloned by us. As a result, we used 50ml induced culture after 3 hours of induction to extract alasan. Overnight lipid precipitation was done by conc. HCl, and then the next morning, we extracted the organic phase by using the solvent extraction method (chloroform: methanol=2:1). The upper aqueous layer was removed, and the lower layer was left to dry overnight. The honey coloured biosurfactant was then seen sticking to the walls of the flask. This was dissolved in deionized water and used for further studies. The working concentration of alasan we used was around 20 mg/ml. 

 <br>
 <br>
 <b> Emulsification Index assay: </b> Emulsification assay was carried out using petrol as it contains both aliphatic and aromatic hydrocarbons. Equal volume of 1:1 of petrol and cell free supernatant obtained after centrifugation of the overnight grown culture was added and mixed by vigorous vortexing for 2 min. The samples were kept for 24 h in a low vibrational area. The emulsification activity of each isolates was calculated by the given formula 
 [1]:Emulsification index= (Height of the emulsion layer/Total height) ‚àó100
 <div>
  <img src="https://github.com/Ravinder210/igem_results_img/blob/main/12result_emulsification%20index%20assay.png?raw=true" alt=" wait for image to be uplaoded"  width="200" height="200">

</div>
<div>
  <img src="https://github.com/Ravinder210/igem_results_img/blob/main/13result_emulsification%20index%20assay2.png?raw=true" alt=" wait for image to be uplaoded"  width="200" height="200">

</div>
 <br>
 <br>
<b>Oil displacement assay:  </b> We first used the crude biosurfactant for performing oil displacement assay with burnt motor oil (https://studio.youtube.com/video/4jjXcuqSytU/edit) ADD THIS VIDEO. We observed an instantaneous clearance zone caused by the recombinant alasan that we produced, thus leading us to infer it is functional (as was previously already noted after our positive EI assay).
<div>
  <img src="https://github.com/Ravinder210/igem_results_img/blob/main/14result_oil%20displacedment%20assay.png?raw=true" alt=" wait for image to be uplaoded"  width="200" height="200">

</div>
<div>
  <img src="https://github.com/Ravinder210/igem_results_img/blob/main/15result_oil%20displacment%20assay_alasan.png?raw=true" alt=" wait for image to be uplaoded"  width="200" height="200">

</div>
<div>
  <img src="https://github.com/Ravinder210/igem_results_img/blob/main/16result_oil%20displacement%20assay_sds.png?raw=true" alt=" wait for image to be uplaoded"  width="200" height="200">

</div>

 <b>Thin Layer Chromatography, or TLC: </b>The dried biosurfactant dissolved in deionized water, was used in performing a qualitative analysis of alasan. We tested for the presence of amino acids, and polysaccharides using ninhydrin spray, p-anisaldehyde and iodine vapors. Alasan tested positive for both amino acids, as well as polysaccharides. 
 <div>
  <img src="https://github.com/Ravinder210/igem_results_img/blob/main/17result_thin%20layer%20chromatography1.png?raw=true" alt=" wait for image to be uplaoded"  width="200" height="200">

</div>
<div>
  <img src="https://github.com/Ravinder210/igem_results_img/blob/main/18result_thin%20layer%20chromatography2.png?raw=true" alt=" wait for image to be uplaoded"  width="200" height="200">

</div>
<div>
  <img src="https://github.com/Ravinder210/igem_results_img/blob/main/19result_thin%20layer%20chromatography3.png?raw=true" alt=" wait for image to be uplaoded"  width="200" height="200">

</div>
 <b> FT-IR: </b> for further confirmation of our TLC results, we decided to carry out IR spectroscopy with the dried sample for alasan. We detected peaks for primary and secondary amines and amides, conjugated alkenes C=C bonds, as well as N-H bond stretching, carboxylic acids. This further correlated to what we knew of the functional groups in the molecule. To the best of our knowledge, this is the first study of FT-IR conducted on alasan. The spectrum has been given below.
 <div>
  <img src="https://github.com/Ravinder210/igem_results_img/blob/main/20result_FTIR.png?raw=true" alt=" wait for image to be uplaoded"  width="200" height="200">

</div>
  <b>ICP-OES: </b> In order to observe the extraction capacity of the biosurfactant on metals, ICP-OES assay was performed. The biosurfactant pellet weighing around 20 mg was dissolved in water with 100 ppm of salt solution. This sample was filtered to remove any contamination. It was then incubated for 12 hours at 200 rpm and 37 C, after which the sample was filtered again to remove the biosurfactant complexes so that only the metal left in the sample is detected. This was compared with the test 100 ppm salt solution which was treated through the same conditions.  

 The assay was performed for three different metals, copper, chromium and cadmium. Alasan shows significant removal for all three metal ions. Copper was found to be reduced by 22%, Chromium 13.7% and Cadmium 8.42%.
 Following are the bar diagrams representing the same.
 
  </p>
  <div>
    <img src="https://github.com/Ravinder210/igem_results_img/blob/main/21result_ICPOES.png?raw=true" alt=" wait for image to be uplaoded"  width="200" height="200">

  </div>
<b>Contact Angle:
</b> 
Contact angle of the biosurfactant alasan was measured and found out to be around 33 degrees. The concentration of the sample was 170 mg/ml. Since it is much less than 90 degrees, it is likely to be a wet table and good biosurfactant.
  </p>
  <div>
    <img src="https://github.com/Ravinder210/igem_results_img/blob/main/22result_contact%20angle.png?raw=true" alt=" wait for image to be uplaoded"  width="200" height="200">

  </div>
<!-- slide show-->
<center>
<style>
* {
  box-sizing: border-box;
}

.column {
  float: left;
  width: 50%;
  padding: 5px;
}

/* Clearfix (clear floats) */
.row::after {
  content: "";
  clear: both;
  display: table;
}
</style>
</head>
<body>

</body>
</html>
</center>
{% endblock %}