{% set ref = 'no' %}
{% extends "layout.html" %}
  
{% block title %}Future Plans{% endblock %}
{% block lead %} The project does not stop here, here is what we are planing to do next!


{% endblock %}
{% block page_content %} 
<h2>Intro</h2>

<p>Over the time that we have worked on this project, we were able to tackle countless challenges using debugging and
   troubleshooting approaches, which allowed us to use our limited time in the most optimal way possible, while also 
   making sure adequate scientific facts support our every decision.</p>  
<p>However, some obstacles were not solved from neither the first, nor second, nor third iteration of
   the <a href="{{ url_for('pages', page='engineering') }}#cycle">Design-Build-Test cycle</a>; a problem-solving 
   approach which led us through the whole project. Which is why in this page we want to discuss our future plans 
   and expand upon plans which we did not get to execute in time.</p>



<h3 >U6PT Design</h3>
<hr>
<h4>Cloning of U6PT</h4>

<p>Initially, when designing the U6PT genetic circuit, we were apprehensive regarding the functional expression of the enzyme, 
  as multiple literature sources stated that expressing plant aromatic prenyltransferases in bacterial systems is not fruitful
   (read more in our <a href="{{ url_for('pages', page='design') }}"> Design </a>page). However, while getting ready to tackle the expression of the gene, we were met with
    hurdles in the cloning process itself. We got undesired results in multiple iterations we did, aiming to pin-point the
     problem. Thus, it was important for our team to brainstorm future plans for achieving successful results. </p>

<p>Our debugging process included tests on the backbone; we have sent it for sequencing and made sure that the desired restriction 
  sites were there, and they have not been mutated. Then, we conducted restriction tests using the enzymes for cloning U6PT 
  (NdeI, NheI). We noticed that the two enzymes work perfectly each one individually, however when assessing their functionality
   together, we were not able to tell if they cut properly, for the reason that the DNA fragment in-between their restriction sites
    is 40 bps, which is too short to appear on the agarose gel. To eliminate the scenario in which the enzymes do not cut together 
    due to steric hinderance, we decided to restrict the backbone in two steps, with each enzyme individually. However, 
    this alteration did not lead to successful cloning results.  </p>
<p>Another approach we took is changing the cloning location on the same backbone, which requires changing the restriction enzymes. 
  This was done by amplifying all U6PT DNA fragments using primers containing overhangs that add the appropriate restriction sites.
   We attempted cloning using the new approach but were presented with undesired results yet again. </p>
<p>This process showed that the backbone we were using, A133-rhlr-tdpp7-mcherry plasmid, is difficult to manipulate; however
   the reasons are still unclear. </p>
<p>To read about our debugging process in detail go to our <a href="{{ url_for('pages', page='engineering') }}#cycle"> 
  Engineering Success </a>page.</p>
<p>As for our future plans, we aim to substantially change the genetic circuit design for U6PT, starting with choosing a new 
  backbone. We will not use the initial design with P2A self-cleaving peptide and mCherry reporter gene downstream to U6PT.
   We will design the circuit with U6PT attached to a histidine tag, all in a regulatory system. </p>



<h4>Functional expression assessment</h4>
<hr>
<p>After tackling the cloning step, we must start the process of evaluating the functional expression of the enzyme. Starting 
  with the transcription stage using qPCR assay. Afterwards, we will continue to evaluate the translation stage, using a poly 
  histidine tag assay. Finally, we aim to build an experiment system to assess enzyme functionality, by introducing its substrate, 
  umbelliferone, and measure the product, 7-demethylsuberosin, using HPLC. </p>



<h4>Transit peptide</h4>
<hr>
<p>According to the results we get from the functionality tests, we can proceed to the transit peptide topic. We can remove
   the transit peptide sequence from the protein sequence, and repeat the steps in the functional expression assessment stage,
    compare the results, and form our conclusions regarding the effects of the transit peptide; We suspect that removing the 
    transit peptide can affect the folding of our enzyme and thus affect its activity and lifetime. </p>

<p>It is important to note that due to inadequate information in the literature regarding expression of plant prenyltransferases 
  in bacterial systems, it is unclear how the transit peptide affects the stability of the mRNA, and thus, positive results in
   qPCR of U6PT might not be fully reliable, as RNA quantification is not indicative of its level of stability. </p>


<h3>XimD & XimE Design</h3>
<hr>
<h4>Enzyme translation</h4>
<p>A continuous step in this circuit would be an expression test for both enzymes to ensure their presence in our system. To 
  fulfill this step, these recombinant proteins must have a poly histidine tag (His tag) fused to their sequence. Then they 
  will be purified using immobilized metal affinity chromatography. Thus, an additional cloning step must be done to insert
   this fragment. Conducting this additional analysis prior to functionality test adds to our engineering process an important 
   trouble shooting point. </p>

<h4>Functionality screening</h4>
<p>Following the expression levels assessments, a functionality test should be run to test the activity of each enzyme 
  as well as determine the kinetic constants of each enzyme especially because they have not been determined yet. During 
  our journey, the most accessible way to test the functionality was to measure the quantity of the enzyme's products using
   HPLC which is an expensive and time-consuming measurement. Our future plans focus on finding a fluorescent analog to each 
   substrate, and testing the kinetic activity based on the fluorescent emission. </p>

<h5>Modelling implementation</h5>
<p>Modelling our biomanufacturing machine system is one of our superior goals in this project, in addition to  the 
  implementation of these models in our wet lab and designs. According to our <a href="{{ url_for('pages', page='model') }}"> Modeling results</a>,
   achieving specific expression ratio between both enzymes in favor of XimE could result in higher biomanufacturing
   yield of our desired metabolite. Many alterations could be done to ensure this ratio, beginning with replacing the
    T7 promoter that regulates the expression of XimD enzyme with a weaker one, or change the RBS sequences for the
     same purpose. </p>


<h3>Esterification reaction</h3>
<hr>
<p>When we reach a point where all cloning designs are successful, and all three enzymes are functionally expressed, 
  we will move forward with the decursin pathway and conduct the final reaction that converts decursinol to decursin.
   This reaction is a non-enzymatic esterification reaction. </p>


<p> Dr. Galit is a researcher in the Schulich faculty of chemistry. After a consultation with her regarding the mentioned 
  rection, we understood that the conversion of decursinol to decursin is chemically done by dissolving decursinol into 
  dichloromethane and adding acyl halide (in our case 3,3-dimethylacryloyl chloride) with sodium bicarbonate as the weak 
  base of our choice (see fig.1). Thus, we expect to get decursin. She also
  offered to conduct this experiment in her lab and under her guidance with the relevant safety equipment and tools.</p>



<img src="https://static.igem.wiki/teams/4179/wiki/future/decursintodecursinol-transparent.png" style="width:80%" class="imgcenter">
<div class="figure-text"><p><b>Figure 1: </b>Decursinol to decursin proposed chemical reaction</p></div>

<h3>OraCell cell line</h3>
<hr>
<p>Visit our <a href="{{ url_for('pages', page='measurement') }}">Measurement</a></li> page to read more about our future plans regarding our new assay.</p>

{% endblock %}
