{% extends "layout.html" %}
  
{% block title %}Measurement{% endblock %}
{% block lead %}Succeeding in manufacturing decursin would be in vain without the proper means of 
detection and quantification. The gold standard for these kinds of measurement is HPLC; a costly, 
time consuming, and expertise requiring method. This led us to develop our own measurement method, OraCell.  {% endblock %}

{% block page_content %}
<h1>Intro</h1>
<hr>
<p>Our project is centered around the production of the secondary metabolite decursin. The system includes the three
    penultimate reactions in the original synthesis pathway of decursin. This involves two enzymatic reactions and
    one chemical reaction. This means, in all, our project would involve four unique metabolites, umbelliferone,
    7-demethylsuberosin, decursinol, and finally decursin. In order to assess if our reactions
    proceeded successfully, we needed a <b>reliable and efficient method of detecting and quantifying our molecules</b> while
    being capable of differentiating between various other metabolites in our system. </p><p> Usually, the state-of-the-art
    method used for such a need is high-performance liquid chromatography (HPLC)<sup>[<a href="#ref1">1</a>]</sup> due to its sensitivity to various
    metabolites and ability to quantify said molecules. However, we quickly realized that while HPLC was a satisfactory
    solution from a technical standpoint, there were many downsides that made HPLC less than optimal. Firstly, we
    had many different designs on the synthetic biology side that would lead to a vast amount of HPLC runs in order
    to compare the various designs and arrive at a conclusion. Secondly, HPLC is a low throughput device, each run
    is time consuming, expensive and the running of the machine requires specific expertise<sup>[<a href="#ref2">2</a>]</sup>. And thirdly, the
    HPLC device itself is very expensive and not widely accessible. These disadvantages deter iGEM groups from pursuing
    worthwhile projects as mentioned on our <a href="{{ url_for('pages', page='human-practices') }}">Human Practices</a> page. </p>
    <p>
    Due to all these reasons,
    we have developed a measurement tool for quantifying metabolites. We named it <b>OraCell</b> since it can assist in giving
    a clearer picture of the future of our project as well as other similar projects. Our method employs <b>Luciferase
    as a reporter gene</b>, and in the presence of the molecule of interest it registers a <b>change in bioluminescence signal.</b>
    As a side note, in Hebrew the word “Or” means light and since we’re utilizing changes in luminescence this made
    the choice of name even more appropriate. </p>
    
<h1>The Hippo Pathway</h1>
<hr>

<p> While researching decursin, we discovered that in addition to being a leading candidate in treatment for chemotherapy-induced
    alopecia, it significantly assists in the inhibition of tumor growth<sup>[<a href="#ref3">3</a>]</sup>. One possible mechanism for its tumor
    suppression capabilities is through the <b>Hippo signaling pathway</b><sup>[<a href="#ref4">4</a>]</sup>. In this pathway, Yap/Taz proteins translocate
    to the nucleus of mammalian cells, bind to a TEAD transcription factor which in turn binds to a specific DNA motif,
    inducing transcription of TEAD-regulated genes<sup>[<a href="#ref5">5</a>]</sup>. In the presence of decursin, the upstream elements in the
    Hippo pathway are phosphorylated, inducing the destruction of Yap/Taz proteins, inhibiting their nuclear localization
    and reducing transcription<sup>[<a href="#ref6">6</a>]</sup>. Following the revelation that decursin impacts an existing signaling pathway,
    we planned to <b>incorporate the Hippo pathway in our method for quantifying and detecting decursin.</b> </p>
<img src="https://static.igem.wiki/teams/4179/wiki/measurment/picture2.png" style="width:57%" class="imgcenter">
<div class="figure-text">
    <p>
        <b>Figure 1: The Hippo pathway</b> (Currey et al. 2021)<sup>[<a href="#ref7">7</a>]</sup> Decursin is thought to interfere with upstream
        elements
    </p>
</div>
<h1>OraCell</h1>
<hr>
<p>The OraCell system is based on an existing plasmid with the Luciferase reporter gene, downstream of TEAD response
    elements (Addgene plasmid <a href="http://n2t.net/addgene:34615">#34615</a>). Transfection of this plasmid
    yields a constitutive expression of the reporter gene. When adding decursin or any metabolite that interacts with
    the Hippo pathway to these cells, the TEAD transcription factor should be unable to bind to the DNA resulting
    in a <b>decrease of the bioluminescence signal.</b> Following consultations with professors knowledgeable in this subject
    in general and this plasmid specifically, we chose to use Chinese Hamester Overian (CHO) cells as the host organism.</p><p> We wanted to create
    a system in which the results will be <b>easily reproducible.</b> Since mammalian cells do not maintain plasmids, a transient
    transfection assay would require a secondary reporter under no regulation, by which the Luciferase signal would
    be normalized. Alternatively, the system could be integrated into the genome of our organism of choice, <b>giving
    rise to a new cell line that is dedicated to researching the Hippo-pathway and the molecules that affect it.</b>
    We decided to go with the latter, and integrated the plasmid into the genome of CHO cells by cloning a selection
    marker that confers resistance to blasticidin, a toxin of mammalian cells, under a CMV promoter. This will reduce
    the noise in our system and remove any unrepeatability between experiments.</p><p> To establish a cell line, CHO cells were transfected with our
    modified system. 24 hours post-transfection we changed the media to a version containing 8 ug/ml blasticidin. Cells
    that do not integrate the plasmid into their genome would dilute the plasmid as they divide, while cells that
    integrated the system would pass the locus down to the daughter cells. After a selection period of 7 days, cells were seeded in a single cell per well ratio using 
    Fluorescence-activated Cell Sorting (FACS). Once propagated, we incubated clonal cell populations in each cell with luciferin, and <b>Luciferase-positive cells were then established as the OraCell system.
    These cells were then capable of determining the concentration of decursin in any given sample.</b> A duration of one hour incubation with decursin was found to be sufficient, the cells are then lysed and release of the existing Luciferase enzyme whose
    amount varies in a <b>decursin-dependent manner.</b> Luciferin is then added, which is catalyzed by the Luciferase enzyme,
    yielding bioluminescence that can be measured with a luminometer. This value can then be compared with a calibration
    curve determining the concentration of purified decursin in the original sample. </p>
<center>
    <div class="row">
        <img src="https://static.igem.wiki/teams/4179/wiki/measurment/oracell-animation.gif" width="85%" height="auto"
            class="imgcenter">
            <div class="figure-text"><p>Figure 2: OraCell example animation</b></p></div>
    </div>
</center>
<br>
<h1>Experiments & Methods</h1>
<hr>
<p>The main pillars of the workflow used to establish the “OraCell” stable cell-line are composed of multiple experiments
    and protocols devised based on knowledge provided by our mentors and our literature review.</p>
<img src="https://static.igem.wiki/teams/4179/wiki/measurment/theoracell-workflow.png" style="width:70%" class="imgcenter">
<div class="figure-text">
    <p>
        <b>Figure 3: The OraCell workflow. </b> 24 hours post-transfection of the OraCell plasmid, cells were
        grown in a selective media that pressured them to integrate the plasmid into their genome. After sorting for
        Luciferase positive cells, these cells are grown and incubated with bacterial-produced decursin. Finally, cells
        are lysed and their Luciferase content can react with luciferin, yielding a measurable luminescence.
    </p>
</div>

<h2>Plasmid Construction</h2>

<p>We used the 8xGTIIC-Luciferase plasmid as a backbone for our plasmid system (Addgene plasmid <a href="http://n2t.net/addgene:34615">#34615</a>). It includes a pGL3b vector and the genes encoding for Luciferase downstream of GTIIC motifs
    with high affinity for TEAD transcription factors. This was convenient due to the abundance of prior results.
    For our stable integration, we required an assembly of a constitutive CMV promoter with a blastR resistance gene
    (iGEM part <a href="http://parts.igem.org/Part:BBa_K1943015">BBa_K1943015</a>) </p>

<h5>Insert 1: CMV promoter</h5>
<p>The constitutive mammalian <a href="http://parts.igem.org/Part:BBa_I712004" target="_blank">CMV promoter</a>, the
    CMV enhancer, and two restriction sites (MluI, HindIII) were amplified from a plasmid provided to us by our PI.
    </p>
<h5>Insert 2: blastR </h5>
<p>The blasticidin resistance gene <a href="http://parts.igem.org/Part:BBa_K1943015" target="_blank">blastR</a> was
    amplified from a plasmid provided to us by our PI's lab. A KpnI site was added using primers we designed, alongside
    the existing HindIII site. </p>
<h5>Assembly</h5>
    <p>The assembly was achieved with restriction of the backbone with mentioned restriction sites. Following the restriction,
        a 3-part ligation (backbone, CMV insert, blastR insert) was performed successfully to achieve the final construct. <a href="{{ url_for('pages', page='measurement') }}#oraora"> Results </a> were validated using sequencing and colony PCR alongside
        gel electrophoresis.</p>    
<div class="row mt-4">
    <div class="col">
        <style>
            * {
                    box-sizing: border-box;
                  }
            
                  .column {
                    float: left;
                    width: 30%;
                    padding: 5px;
                  }
            
                  /* Clearfix (clear floats) */
                  .row::after {
                    content: "";
                    clear: both;
                    display: table;
                  }
            
        </style>
        <style>
            .figure-text {
                    font-size: 12px;
                    color: #787277;
                  }
            
        </style>
        <div class="row justify-content-center">
            <div class="col-3">
                <img src="https://static.igem.wiki/teams/4179/wiki/measurment/pic4new.png" alt="Pcpt" style="width: 100%" class="imgcenter">                
                <div class="figure-text">
                    <p>
                        <b>(A)</b>
                    </p>
                </div>
            </div>
            <div class="col-3">
                <img src="https://static.igem.wiki/teams/4179/wiki/measurment/pic5new.png" alt="Pspt1" style="width:100%" class="imgcenter">                
                <div class="figure-text">
                    <p>
                        <b>(B)</b>
                    </p>
                </div>
            </div>
            <div class="col-3">
                <br>
                <img src="https://static.igem.wiki/teams/4179/wiki/measurment/pic6new.png" alt="Fcpt1" style="width:102%" class="imgcenter">                
                <div class="figure-text">
                    <p>
                        <b>(C)</b>
                    </p>
                </div>
            </div>
            <div class="col-3">
                <br>
                <img src="https://static.igem.wiki/teams/4179/wiki/measurment/2-luc-blast-oracell.png" alt="Fcpt1" style="width:102%" class="imgcenter">                
                <div class="figure-text">
                    <p>
                        <b>(D)</b>
                    </p>
                </div>
            </div>
        </div>
        <div class="figure-text">
            <p>
                <b>Figure 4: Plasmids used for the construction of the OraCell system.</b> (A) The backbone derived from 8xGTIIC-Luciferase plasmid
                (B) The insert amplified from the pRGEN-CMV-CjCAS9 plasmid
                (C) The insert amplified from the pTRE backbone
                (D) Final construct plasmid containing BlastR, CMV promoter, and Luciferase enzyme downstream
                    of 8xGTIIC motifs that was transfected into CHO cells
            </p>
        </div>    
    </div>
</div>

<h2>Preliminary calibration experiments </h2>

<p>Once we verified that our plasmid was ready to be transfected to the mammalian cells, we were able to continue
    to the next stage in the workflow.</p><p> We chose to work with CHO cells, because they are easily
    growable in large-scale cultures and demonstrate high stability and safety profiles.</p><p> Working with mammalian cells in the time
    span of this project required delicate planning. In accordance, preliminary experiments were conducted to get
    acquainted with the protocols of splitting, seeding and transfection, and find optimal parameters such as the
    time it takes for plates to reach confluence and the optimum concentration of the transfection reagents . </p>

<h4>Seeding calibration</h4>
<p>CHO cells were grown in 10cm plates, and experiments were performed in 6-well plates. Due to time constraints,
    an optimal seeding density of a 6-well plate was sought, such that overnight incubation would result in a confluent
    plate on the next day.</p><p> 6-well plates were seeded with a different cell density ranging from 10<sup>4</sup> cells /cm<sup>2</sup> to
    10<sup>5</sup> cells/cm<sup>2</sup>. After overnight incubation we counted the number of cells in each plate using a counting chamber.
    </p>

<h4>Linear Polyethylenimine (PEI, Mw 25kDa) Calibration</h4>
<p>Our chosen method of transfection was PEI. This transfection reagent is used to introduce the plasmid DNA to the
    host cell, it condenses DNA into positively charged particles that are endocytosed and transferred to the cytoplasm.
   <sup>[<a href="#ref8">8</a>]</sup> PEI was chosen due to its availability and low cost. Stock solution concentration was 1mg/ml.</p><p>
    We conducted an experiment to find the optimum mass of PEI in transfection experiments. While PEI provides a way to introduce
    DNA into the cell, it is also toxic. An optimal amount would be high enough to transfect the highest percentage
    of cells possible, while retaining a reasonable percentage of viable cells.</p><p> Our experiment included a plasmid
    backbone with a <a href="http://parts.igem.org/Part:BBa_K592100" target="_blank">Blue Fluorescent Protein (BFP) insert</a>    reporter gene. The BFP would be used to quantify the efficiency of the transfection; cells that underwent transfection
    would express BFP that in turn expresses fluorescence.</p><p></p> 2.5 μg of BFP plasmid was mixed with increasing amounts
    of PEI and used FACS to measure the cell’s viability and fluorescence 48 hours post-transfection. </p>

<h2>Stable Integration</h5>
<p>To establish a stable clonal cell line, cells were transfected with the blasticidin resistance gene (BlastR)-modified
    Luciferase plasmid and subjected to a high concentration of blasticidin. Then, we used FACS (Bigfoot Spectral
    Cell Sorter) to perform single-sorting cell into individual wells on a plate and allowed cells to proliferate
    before testing their Luciferase expression with luciferin.</p>

<h2>Luciferase Assay</h2>
<p>To measure the transfected cell reporter gene, Luciferase, we performed a luminescence assay. This was done both
    to validate the transfection itself, and to study how decursin affects the gene expression.</p><p> We used a Biotium’s
    Firefly Luciferase Assay Kit 2.0, cells were lysed, and luciferin was added to the lysate. This resulted in an
    oxidation reaction that emitted a measurable signal. The signal was measured using the bioluminescent mode of
    a plate reader. Our positive control for the experiment was <I>E. coli</I> β-10 cells transfected with a p.lux plasmid
    that contains the genes encoding for both Luciferase and luciferin. </p>

<h1>Results</h1>
<hr>

<p> Our initial cloning, calibrations and luminescence experiments can be found on our <a href="{{ url_for('pages', page='results') }}">results</a> page.
     Following these preliminary results, we began adding decursin and measuring the luminescence to investigate
    if there was indeed a decrease. The first of these experiments was a time calibration experiment to see under
    which incubation time we get an optimal decrease in luminescence. The two negative controls used in the experiment,
    OraCell cells without luciferin and CHO cells that don’t express Luciferase, both displayed minimal luminescence
    in comparison to the positive control and experiment group (Figure 5). Additionally, the positive control, OraCell
    cells without decursin, following 22 hours incubation showed higher luminescence compared to OraCell cells that
    were incubated with decursin. This was a good indication that the system functioned as expected, in the presence
    of decursin, there was a decrease in the transcription of our reporter gene.</p>
<center>
<img src="https://static.igem.wiki/teams/4179/wiki/measurment/decursin-time-calibration.png" width="50%">
<div class="figure-text justify-content-center">
    <p>
        <b>Figure 5 </b>. Decursin incubation time calibration. OraCell cells were incubated with decursin with various time points ranging from 1-22 hours, lysed, luciferin was added and bioluminescence was measured. In addition, there were two types of negative control used, OraCell cells without luciferin added and CHO cells that do not express Luciferase. The positive control was OraCell cells incubated without decursin
    </p>
</div>
</center>

<p> This implied that with further testing
    of decursin concentrations, it would be possible to build a calibration curve of decursin concentrations as a
    function of luminescence. To our surprise, we noticed an unexpected trend in the experiment group, a periodic
    oscillation. The level of luminescence was low following a one-hour incubation and gradually increased until the
    peak around 12 hours, at which point the luminescence begins decreasing until it reaches a level roughly equivalent
    to the starting luminescence. We speculated that this could be due to the fact that the Hippo pathway is involved
    in cell proliferation and naturally oscillates during the cell cycle<sup>[<a href="#ref9">9</a>]</sup>. This led us to pursue a two-fold experiment
    which consisted of the testing of various decursin concentrations at two different time points as well as the
    highest and zero concentrations at two additional time points. </p><p> Similar to the previous experiment (Figure 5),
    the OraCell cells incubated with decursin once again showed a periodic behavior (Figure 6). Following one hour
    of incubation time with decursin, the greatest decrease in luminescence in comparison to the cells
    without decursin was observed. The differences between the control group, without decursin, and the cells that were incubated
    with decursin gradually decreases until their luminescence was roughly equivalent following a ten-hour incubation,
    meaning that decursin had a minimal impact. This result fits nicely with the previous finding, that following
    a 12-hour incubation with decursin, we receive a minimal decrease in luminescence (Figure 5). An interesting phenomenon
    occurs following 16 hours of incubation with and without decursin. In the control group for the first ten hours,
    the luminescent signal stayed approximately the same but after 16 hours, there’s a marked decrease (Figure 5).
    Even more interesting is the fact that the OraCell cells following a 16-hour incubation with decursin, show a
    higher luminescence signal compared to the control. This indicates that decursin caused up-regulation of the
    expression of the Luciferase enzyme, contrary to our predicted mechanism. This point is illustrated further by
    looking at the calibration curve of the same experiment (figure 7). </p>

    <center>
<img src="https://static.igem.wiki/teams/4179/wiki/measurment/decursin-16-hour-time-experiment.png" width="30%">
<div class="justify-content-center figure-text">
    <p>
        <b>Figure 6: Decursin incubation time.</b> OraCell cells were incubated with and without decursin at various
        time points ranging from 1-16 hours, lysed, luciferin was added, and bioluminescence was measured. The cells
        without decursin served as the positive control, to see how much decursin impacted the cells’ bioluminescence.
        
    </p>
</div>
</center>

<p>The calibration curve experiment was conducted on two of the time points from the incubation time experiment, 3
    and 16 hours. The OraCell cells were incubated with six different decursin concentrations, 0, 15, 30, 60, 120
    and 240 µM for both time points. For the three-hour incubation, it is difficult to define a pattern. It appears
    that decursin causes mild fluctuations in the luminescence, both increasing and decreasing, compared to the
    cells without decursin, the zero concentration (Figure 7). The 16-hour incubation, on the other hand, showed a
    very clear trend, of higher luminescence with increasing decursin concentrations. Saturation was observed at the 60 µM
    concentration, at which point the decursin no longer observed to have an effect. Just as
    with the incubation time experiment, we see the opposite effect of what was expected. We expected that with increased
    decursin concentrations, luminescence will decrease; however, luminescence increased. </p>
<center>
<img src="https://static.igem.wiki/teams/4179/wiki/measurment/calibration-curve-decursin-3-and-16-hour-experiment.png" width="40%">
<div class="figure-text">

    <p>
        <b>Figure 7: Dose response for decursin concentrations at 3 and 16 hours.</b> OraCell cells
        were incubated with six concentrations of decursin ranging from 0-240 µM for two time points, 3 and 16 hours.
        The cells were then lysed, luciferin was added, and bioluminescence was measured. The luminescence was normalized
        to the zero concentration of decursin for each time point, respectively. Note that for the 16 h dose response, the trend shows a linear dependency.
    </p>
</div>
</center>

<p>Our working hypothesis to explain this phenomenon is that originally, we only partially understood decursin’s impact
    on the Hippo pathway. The Hippo pathway functions in two modes, <b>Hippo on</b> and <b>Hippo off</b> and these are both controlled
    by most of the same proteins<sup>[<a href="#ref11">10</a>]</sup>. Thus, if decursin is capable of interfering with one mode of this pathway, it’s
    reasonable to assume that it can then interfere with the second mode as well. This means that if the OraCell cells
    are currently in the Hippo off mode, there are many upstream elements that interact with the Yap/Taz proteins,
    causing them to be dephosphorylated and cause expression<sup>[<a href="#ref10">11</a>]</sup>. However, if the OraCell cells are currently in
    the Hippo on mode, there would also be other upstream elements that interact with the Yap/Taz proteins, causing them to be
    phosphorylated and inhibit expression<sup>[<a href="#ref10">11</a>]</sup>.</p><p>
      
    This would also explain why there’s a decrease in Luciferase expression
    in cells that didn’t receive decursin following the 16-hour incubation (Figure 6). The cells have made the switch
    to Hippo on, causing a natural decrease in expression, and therefore, when decursin is added, this process is
    interrupted giving an increase in Luciferase expression. This would also explain the unclear trend found following
    the three-hour incubation (Figure 7). It’s possible that the Hippo pathway is found in a limbo stage, between
    on and off, leading to conflicting signals and thus, decursin has a fluctuating effect. This phenomenon can be
    further investigated by adding more time points and adding changing decursin concentrations at each point as well
    as monitoring the Luciferase expression at each time point without the addition of decursin. Altogether, this would give a
    clearer picture of the natural fluctuations in Luciferase expression as a result of the Hippo pathways without
    decursin and if decursin is truly able to interfere with the Hippo pathway, regardless of the mode.</p>

<h1 id="future"> Prospects </h1>
<hr>

<p>The OraCell assay was shown to be a efficient, simple and high-throughput method that holds the potential of detecting
    and quantifying a variety of molecules. The method still needs a few more experiments to verify its capabilities,
    which should hopefully be accomplished in the coming weeks to show its potential utility for groups all across
    the world.</p><p>
    
    This measurement could be repeated by other iGEM teams by preforming a simple Luciferase assay on the
    OraCell cell line. The protocols used were described in our <a href="{{ url_for('pages', page='Protocols') }}">Protocols</a> page.</p>
    
    <p>In future implementations, the
    OraCell cell line could be used not only as for quantification assays by other teams, but also to mass screen molecules
    for their effect on the Hippo Pathway. Compared to traditional YAP/TAZ assays that require re-transfection on
    each assay, OraCell provides a high-throughput and low-cost solution. Generating datasets of molecules related to
    the Hippo Pathway could have uses alongside machine learning algorithms for new drug discovery. The 
    research of the Hippo Pathway, an important tumor suppressor<sup>[<a href="#ref11">12</a>]</sup>, could lead to advances in the field of cancer
    research and open the door for new treatment strategies. </p> {% endblock %}
    
    
{% block references %}

  <ol class="reftext">
    <li id="ref1">
    Higgs, R. E., Zahn, J. A., Gygi, J. D., & Hilton, M. D. (2001). Rapid Method To Estimate the Presence of Secondary
    Metabolites in Microbial Extracts. Applied and Environmental Microbiology, 67(1), 371. <a href=" https://doi.org/10.1128/AEM.67.1.371-376.2001"
    target="_blank"> https://doi.org/10.1128/AEM.67.1.371-376.2001</a>
    
    <li id="ref2">
    Perea, S., Pennick, G. J., Modak, A., Fothergill, A. W., Sutton, D. A., Sheehan, D. J., & Rinaldi, M. G. (2000).
    Comparison of High-Performance Liquid Chromatographic and Microbiological Methods for Determination of Voriconazole
    Levels in Plasma. Antimicrobial Agents and Chemotherapy, 44(5), 1209. <a href="https://doi.org/10.1128/AAC.44.5.1209-1213.2000"
    target="_blank">https://doi.org/10.1128/AAC.44.5.1209-1213.2000</a>
   
    <li id="ref3">
   Ge, Y., Yoon, S. H., Jang, H., Jeong, J. H., & Lee, Y. M. (2020). Decursin promotes HIF-1a proteasomal degradation
    and immune responses in hypoxic tumour microenvironment. Phytomedicine, 78, 153318. <a href="https://doi.org/10.1016/J.PHYMED.2020.153318"
    target="_blank"> https://doi.org/10.1016/J.PHYMED.2020.153318</a>
   
    <li id="ref4">
    Li, J., Wang, H., Wang, L., Tan, R., Zhu, M., Zhong, X., Zhang, Y., Chen, B., & Wang, L. (2018). Decursin
    inhibits the growth of HepG2 hepatocellular carcinoma cells via Hippo/YAP signaling pathway. Phytotherapy
    Research : PTR, 32(12), 2456-2465. <a href="https://doi.org/10.1002/PTR.6184" target="_blank">https://doi.org/10.1002/PTR.6184</a>                        
    
    <li id="ref5">
    Boopathy, G. T. K., & Hong, W. (2019). Role of Hippo Pathway-YAP/TAZ signaling in angiogenesis. Frontiers
    in Cell and Developmental Biology, 7(APR), 49. <a href="https://www.sigmaaldrich.com/NL/en/product/sigma/sml0786?gclid=Cj0KCQjwyt-ZBhCNARIsAKH1176_PfbFoii-r4Gj7L0CHfNBOxnYuuQAd81No8mwyswxgiKjf1eZ14EaAuC8EALw_wcB&gclsrc=aw.dshttps://doi.org/10.3389/FCELL.2019.00049/BIBTEX "
    target="_blank"> https://doi.org/10.3389/FCELL.2019.00049/BIBTEX </a>
   
    <li id="ref6">Allegra, A., Pioggia, G., Innao, V., Musolino, C., & Gangemi, S. (2021). New Insights into YES-Associated
    Protein Signaling Pathways in Hematological Malignancies: Diagnostic and Therapeutic Challenges. Cancers,
    13(8). <a href="https://doi.org/10.3390/CANCERS13081981" target="_blank">https://doi.org/10.3390/CANCERS13081981</a>
    <li id="ref7"> Currey, L., Thor, S., & Piper, M. (2021). TEAD family transcription factors in development and disease.
    Development (Cambridge), 148(12). https://doi.org/10.1242/DEV.196675/269158
    <li id="ref8">Longo, P. A., Kavran, J. M., Kim, M. S., & Leahy, D. J. (2013). Transient mammalian cell transfection with
    polyethylenimine (PEI). In Methods in enzymology (Vol. 529, pp. 227-240). Academic Press.
    <li id="ref9"> Kim, W., Cho, Y. S., Wang, X., Park, O., Ma, X., Kim, H., Gan, W., Jho, E. hoon, Cha, B., Jeung, Y.
    ji, Zhang, L., Gao, B., Wei, W., Jiang, J., Chung, K. S., & Yang, Y. (2019). Hippo signaling is intrinsically
    regulated during cell cycle progression by APC/CCdh1. Proceedings of the National Academy of Sciences
    of the United States of America, 116(19), 9423–9432.
    <li id="ref10"> Johnson, R., & Halder, G. (2014). The two faces of Hippo: targeting the Hippo pathway for regenerative medicine and cancer treatment. Nature reviews Drug discovery, 13(1), 63-79.
    <li id="ref11"> Noorbakhsh, N., Hayatmoghadam, B., Jamali, M., Golmohammadi, M., & Kavianpour, M. (2021). The Hippo
    signaling pathway in leukemia: function, interaction, and carcinogenesis. Cancer Cell International
    2021 21:1, 21(1), 1–11. https://doi.org/10.1186/S12935-021-02408-7
    <li id="ref12"> Liu, Q., Liu, X., & Song, G. (2021). The Hippo Pathway: A Master Regulatory Network Important in
        Cancer. Cells, 10(6), 1416. https://doi.org/10.3390/cells10061416
  </ol>


{% endblock %}
